### DISCUSSION TITLE
KEDA Chart: v2.15.1

### DISCUSSION THREAD

**Discussion by JorTurFer** [kedacore/charts#677] [Announcements]
Chart: v2.15.1 | App: v2.15.1

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.15.1 -n keda --create-namespace
```

### Fixes/Changes

- Support to KEDA v2.15.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.15.1'>KEDA Chart: v2.15.1</a>.</em>

**Comment by jdinsel-xealth** [comment]
`permissions.operator.restrict.secret` when set to true seems to result in an inoperable deployment. https://github.com/kedacore/charts/issues/685
